packages/block-library/src/paragraph/editor.scss

```diff
@@ -15,3 +15,28 @@
 		opacity: 0;
 	}
 }
+
+
+// Precise partial text selections.
+// @todo: For now, proof of concept. But code needs to be moved to the most appropriate place.
+// Furthermore, this code right now is likely going to affect numerous nested blocks, so
+// it will be important to scope it right!
```

Comment:
What blocks will be negatively affected? I can't think of any, so maybe no need to scope?

Reply:
It's important to test. This rule is the one we should be sure about:

```
.block-editor-block-list__layout {
	user-select: none;
}
```

Basically it says, anything that's inside `.block-editor-block-list__layout`, you can't select, unless we explicitly re-enable it using `user-select: text;`, or even `user-select: auto;`